pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -1436,7 +1435,7 @@ func isAutoPassthrough(obj config.Config, l k8s.Listener) bool {
 	}
 	expectedPort := "15443"
 
-	if port, f := obj.Labels[annotation.NetworkingGatewayPort.Name]; f {
+	if port, f := obj.Labels[label.NetworkingGatewayPort.Name]; f {
```

Comment:
This change is not related to tunneling. I had to change this line, because `NetworkingGatewayPort` was moved to another package.